getting-started/kubernetes/self-managed-public-cloud/aws.md

```diff
@@ -80,8 +80,11 @@ The geeky details of what you get:
 
 {% include open-new-window.html text='Kubespray' url='https://kubespray.io/' %} is a tool for provisioning and managing Kubernetes clusters with support for multiple clouds including Amazon Web Services. {{site.prodname}} is the default networking provider, or you can set the `kube_network_plugin` variable to `calico`. See the {% include open-new-window.html text='Kubespray docs' url='https://kubespray.io/#/?id=network-plugins' %} for more details.
 
-### Above and beyond
+### Next steps
 
-- {% include open-new-window.html text='Video: Everything you need to know about Kubernetes pod networking on AWS' url='https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-pod-networking-on-aws/' %}
+**Required**
 - [Install and configure calicoctl]({{site.baseurl}}/getting-started/clis/calicoctl/install)
+
+**Recommended**
+- {% include open-new-window.html text='Video: Everything you need to know about Kubernetes pod networking on AWS' url='https://www.projectcalico.org/everything-you-need-to-know-about-kubernetes-pod-networking-on-aws/' %}
```

Comment:
Window/tab opening templating seems a bit inconsistent. There are a few instances where we do not use include open-new-window.html but we are using them in a few other places. We should perhaps keep things the same - guessing since it's linked to the www.projectcalico.org, the behavior we want is opening things in a new tab?

Reply:
Good catch. Alex and I never really discussed links to projectcalico, but it seems the right call to be separate window.